AppCenter/AppCenterTests/MSEncrypterTests.m

```diff
@@ -17,80 +19,411 @@ @implementation MSEncrypterTests
 - (void)setUp {
   [super setUp];
   self.keychainUtilMock = [MSMockKeychainUtil new];
-  self.keyTag = @"kMSTestEncryptionKeyTag";
 }
 
 - (void)tearDown {
   [self.keychainUtilMock stopMocking];
-  [MSEncrypter deleteKeyWithTag:self.keyTag];
+  [MSMockKeychainUtil clear];
 }
 
-- (void)testEncryption {
+- (void)testEncryptWithCurrentKey {
 
   // If
-  MSEncrypter *encrypter = [[MSEncrypter alloc] initWitKeyTag:self.keyTag];
-  NSString *stringToEncrypt = @"Test string";
+  NSString *clearText = @"clear text";
+  NSString *keyTag = kMSEncryptionKeyTagAlternate;
+  NSString *expectedMetadata = [NSString stringWithFormat:@"%@/AES/CBC/PKCS7/32", keyTag];
+
+  // Save metadata to user defaults.
+  MSMockUserDefaults *mockUserDefaults = [MSMockUserDefaults new];
```

Comment:
Use property for this mock. You can see an example in other test classes that have `self.settingsMock`.

Reply:
We sometimes use properties for mocks and we sometimes don't. Why is there a preference for one way or the other?

Reply:
If it is used by a few tests, we don't need it to be a property. In your case, it is used by all tests except one.

Reply:
Got it. Fixed.